index.d.ts

```diff
@@ -158,5 +162,36 @@ export function likelyWithKeyboard(window: MinimalWindow): boolean
 export function getHotKeyHint(
   window: MinimalWindow,
   code: string,
-  overrides?: HotkeyOverride
+  transformers?: Transformer[]
 ): string
+
+/**
+ * Provides a transformer for hotkey overrides that can be used
+ * with `hotkeyKeyUX`.
+ *
+ * ```js
+ * import { startKeyUX, hotkeyKeyUX, overrides } from 'keyux'
+ *
+ * startKeyUX(window, [
+ *   hotkeyKeyUX([overrides({
+ *     'alt+b': 'b' // Override B to Alt + B
+ *   })])
+ * ])
+ * ```
+ */
+export function overrides(config: HotkeyOverride): Transformer
+
+/**
+ * Provides a transformer for hotkey overrides that can be used
+ * with `getHotKeyHint`. Use the same override config in `hotkeyKeyUX`
+ * and `getHotKeyHint` to get accurate hints.
+ *
+ * ```js
+ * import { getHotKeyHint, hintOverrides, hotkeyKeyUX, overrides } from "keyux"
+ *
+ * let config = { 'alt+b': 'b' }
+ * hotkeyKeyUX([overrides(config)]) // Override B to Alt + B
+ * getHotKeyHint(window, 'b', [hintOverrides(config)]) // Alt + B
+ * ```
+ */
+export function hintOverrides(config: HotkeyOverride): Transformer
```

Comment:
It is very easy to user to send `overrides` instead of `hintOverrides`.

One solution is to add protection with types adding 2 different `Transformer` types.

Another option is to make transformers both-way. I like this idea because seems like we will need `hotkey`/`hint` versions of transformer for every transformer.

Reply:
Yes, let's go with both-way transformers.